docs/en/ingest-management/elastic-agent/configuration/providers/kubernetes-provider.asciidoc

```diff
@@ -331,6 +331,81 @@ The policy generated by this configuration looks like:
         target: orchestrator
 ----
 
+[discrete]
+== Autodiscover targeted Pods using hints (standalone mode)
+
+Standalone Elastic Agent supports autodiscover based on hints from the provider.
+The hints system looks for hints in Kubernetes Pod annotations which have the prefix `co.elastic.hints`.
```

Comment:
what `hints system` here means?

Reply:
Maybe hints mechanism instead of system.

Reply:
👍🏼